DESCRIPTION

```diff
@@ -26,4 +28,6 @@ Imports:
     MASS
 RoxygenNote: 7.1.1
 Config/testthat/edition: 3
-License: GPL (>= 3)
+License: GPL-3
```

Comment:
Other Pfizer Open Source work is now using the MIT license rather than GPL-3.

Reply:
Done, Switched the GPL license to the MIT license.